components/sysutils/procps/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2019 Michal Nowak
+#
+
+PREFERRED_BITS=		64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		procps
+COMPONENT_VERSION=	3.3.15
+COMPONENT_SUMMARY=	Linux ps utilities
+COMPONENT_PROJECT_URL=	https://gitlab.com/procps-ng/procps
+COMPONENT_FMRI=		utilities/procps
```

Comment:
at least utility/procps to be consistent, perhaps, better keep existing name as it just the same watch?